istioctl/cmd/root.go

```diff
@@ -72,6 +72,9 @@ func defaultLogOptions() *log.Options {
 	o.SetOutputLevel("processing", log.ErrorLevel)
 	o.SetOutputLevel("source", log.ErrorLevel)
 	o.SetOutputLevel("analysis", log.WarnLevel)
+	o.SetOutputLevel("kubectlcmd", log.WarnLevel)
+	o.SetOutputLevel("installer", log.WarnLevel)
```

Comment:
Oh thats nice. So its warn for istioctl but in operator mode we see all the logs like normal?

Maybe outside the scope here, but if I run:
```
$ go run ./istioctl/cmd/istioctl manifest apply -y --logtostderr
- Applying manifest for component Base...
✔ Finished applying manifest for component Base.
- Applying manifest for component Pilot...
✔ Finished applying manifest for component Pilot.
- Applying manifest for component Addon...
- Applying manifest for component IngressGateways...
- Applying manifest for component Telemetry...
- Pruning objects for disabled component Citadel...
✔ Finished pruning objects for disabled component Citadel.
✔ Finished applying manifest for component Addon.
✔ Finished applying manifest for component IngressGateways.
✔ Finished applying manifest for component Telemetry.
2020-01-23T19:52:13.866766Z     info

✔ Installation complete

```

There is a random `info` line. Not a big deal though. Maybe just means one of them is missing a scope?

Reply:
Yes, WARN for `istioctl`, INFO for Operator mode.  Unfortunately I only tested my changes with `--dry-run`.  Good catch on the rest; currently I am blocked because I can't `apply`.  I have added `do-not-merge` while I sort that problem out.

Reply:
I am fine with merging as is, its strictly an improvement. But maybe we need to remove the --logtostderr option now (and make it always enabled) that we have this? Since in the normal case we won't have any logs